ingestion/src/metadata/ingestion/source/database/postgres_query_parser.py

```diff
@@ -146,7 +146,7 @@ def process_table_query(self) -> Optional[Iterable[TableQuery]]:
                     rows = conn.execute(
                         self.get_sql_statement(
                             start_time=self.start + timedelta(days=i),
-                            end_time=self.start + timedelta(days=i + 2),
+                            end_time=self.end + timedelta(days=i),
```

Comment:
I believe this should be `self.start` only because on the beginning of the loop we are calculating the day difference and basically traversing from start time to end time putting `self.end` will violate the logic